tests/test_runner.py

```diff
@@ -15,7 +15,10 @@
 # You should have received a copy of the Apache License
 # along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
 
+import logging
 import os
+import subprocess
+from types import SimpleNamespace
 
 import pytest
 from pytest_mock import mocker  # noqa F401
```

Comment:
Random note... the setup.cfg has a require for 'pytest_mock' but the 'official' package name is 'pytest-mock' (https://pypi.org/project/pytest-mock/) . Doesn't really matter but just noticed.

Reply:
Is this import needed?  In theory pytest-mock installs a pytest plugin that provides a fixture that should get discovered when using the pytest runner.